docs/integrations/observability/_index.rst

```diff
@@ -242,7 +270,7 @@ Health Status
 
 Determined master reports a metric, ``determined_healthy``, with value ``1`` when major dependencies
 are reachable and ``0`` otherwise. Visit `alerting
-<https://github.com/determined-ai/determined/blob/observability_feature_branch/docs/integrations/prometheus/_index.rst#alerts>`__
+<https://github.com/determined-ai/determined/blob/main/docs/integrations/prometheus/_index.rst#alerts>`__
```

Comment:
please replace this "https" link with the following:

To learn how to set up alerts, visit :ref:`kubernetes-observability`.

that won't work until you place the following code at the very top (line 1) of the integrations/observability/_index.rst page:

.. _kubernetes-observability:

Then run `make fmt` and build docs locally and they should build without error.

Reply:
(++ https://www.sphinx-doc.org/en/master/usage/referencing.html)